assets/price-per-item.js

```diff
@@ -0,0 +1,109 @@
+        this.input.addEventListener('change', this.onInputChange.bind(this));
+
+        this.getVolumePricingArray();
+      }
+
+      updatePricePerItemUnsubscriber = undefined;
+      cartUpdateStartedUnsubscriber = undefined;
+      variantIdChangedUnsubscriber = undefined;
+
+      connectedCallback() {
+        // Update varinatId if variant is switched on product page
+        this.variantIdChangedUnsubscriber = subscribe(PUB_SUB_EVENTS.variantChange, (event) => {
+          this.variantId = event.data.variant.id.toString();
+          this.getVolumePricingArray();
+        });
+        this.cartUpdateStartedUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdateStarted, (event) => {
+          if (event.variantId !== this.variantId) return;
+          this.querySelector('.loading-overlay').classList.remove('hidden');
+        });
+
+        this.updatePricePerItemUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdate, (response) => {
+          this.querySelector('.loading-overlay').classList.add('hidden');
+          // Item was added to cart via product page
+          if (response.cartData['variant_id'] !== undefined) {
+            this.updatePricePerItem(response.cartData.quantity);
+          // Qty was updated in cart
+          } else if (response.cartData.item_count !== 0) {
+            const isVariant = response.cartData.items.find((item) => item.variant_id.toString() === this.variantId);
+            if (isVariant) {
+              // The variant is still in cart
+              this.updatePricePerItem(isVariant.quantity);
+            } else {
+              // The variant was removed from cart, qty is 0
+              this.updatePricePerItem(0);
+            }
+          // All items were removed from cart
+          } else {
+            this.updatePricePerItem(0);
+          }
+        });
+      }
+
+      disconnectedCallback() {
+        if (this.updatePricePerItemUnsubscriber) {
+          this.updatePricePerItemUnsubscriber();
+        }
+        if (this.cartUpdateStartedUnsubscriber) {
+          this.cartUpdateStartedUnsubscriber();
+        }
+        if (this.variantIdChangedUnsubscriber) {
+          this.variantIdChangedUnsubscriber();
+        }
+      }
+
+      onInputChange() {
+        this.updatePricePerItem();
+      }
+
+      updatePricePerItem(updatedCartQuantity) {
+        // updatedCartQuantity is undefined when qty is updated on product page. We need to sum entered qty and current qty in cart.
+        if (updatedCartQuantity === undefined) {
+          const enteredQty = parseInt(this.input.value);
+          this.currentQtyForVolumePricing = this.getCartQuantity(updatedCartQuantity) + enteredQty;
+        // updatedCartQuantity is not undefined when qty is updated in cart. We need to sum qty in cart and min qty for product.
+        } else {
+          this.currentQtyForVolumePricing = this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.min);
+        }
```

Comment:
```suggestion
 const enteredQty = parseInt(this.input.value);

this.currentQtyForVolumePricing = updatedCartQuantity ? this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.min) : this.getCartQuantity(updatedCartQuantity) + enteredQty;
```

Reply:
I think you could try

`this.currentQtyForVolumePricing = updatedCartQuantity === undefined ? (this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.min)) : (this.getCartQuantity(updatedCartQuantity) + enteredQty);`

Reply:
this should work :)